src/style/blocks/card.css

```
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  margin: 0 auto;
+
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-top: 40px;
+}
+
+.title {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-text-color);
+}
+
+.card__code {
+  margin-top: 4px;
+}
+
+.code {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: var(--second-text-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.price-title {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--second-text-color);
+}
+
+.price-number {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-text-color);
+}
+
+.card__btn {
+  margin-top: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.btn {
+  height: 40px;
+
+  background: var(--btn-color);
+  border-radius: 5px;
+
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: white;
+
+  transition: 0.3s all ease;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```